### ISSUE TITLE
Impossible to install (npm shasum check failed)

### ISSUE THREAD

**Issue by maur1th** [hitgeek/simple-hl7#16]
Not sure if I should send this issue to npm repo or here. Tried to install the package, got this error:

```npm ERR! shasum check failed for /var/folders/3l/hc52g2s91svgrmb8zzbg4sf00000gv/T/npm-909-3f0ea4b9/registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz
npm ERR! Expected: bc24c25132a3ca7f4dbdef96dfb9ed511f89746b
npm ERR! Actual:   4594969f78105fb375e0879c0507ddba9a662141
npm ERR! From:     https://registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz```

Full log:
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'install',
1 verbose cli   'simple-hl7' ]
2 info using npm@2.14.12
3 info using node@v4.2.4
4 verbose install initial load of /Users/tm/Code/clicrdv-gcal/package.json
5 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/async/package.json
6 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/body-parser/package.json
7 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/ejs/package.json
8 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/express/package.json
9 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/googleapis/package.json
10 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/istanbul/package.json
11 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/jasmine/package.json
12 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/jsonwebtoken/package.json
13 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/moment/package.json
14 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/moment-timezone/package.json
15 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/mysql/package.json
16 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/request/package.json
17 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/rrule/package.json
18 verbose installManyTop reading scoped package data from /Users/tm/Code/clicrdv-gcal/node_modules/sequelize/package.json
19 info package.json rrule@2.1.0 No license field.
20 verbose readDependencies loading dependencies from /Users/tm/Code/clicrdv-gcal/package.json
21 silly cache add args [ 'simple-hl7@^1.4.2', null ]
22 verbose cache add spec simple-hl7@^1.4.2
23 silly cache add parsed spec Result {
23 silly cache add   raw: 'simple-hl7@^1.4.2',
23 silly cache add   scope: null,
23 silly cache add   name: 'simple-hl7',
23 silly cache add   rawSpec: '^1.4.2',
23 silly cache add   spec: '>=1.4.2 <2.0.0',
23 silly cache add   type: 'range' }
24 silly addNamed simple-hl7@>=1.4.2 <2.0.0
25 verbose addNamed ">=1.4.2 <2.0.0" is a valid semver range for simple-hl7
26 silly addNameRange { name: 'simple-hl7', range: '>=1.4.2 <2.0.0', hasData: false }
27 silly mapToRegistry name simple-hl7
28 silly mapToRegistry using default registry
29 silly mapToRegistry registry https://registry.npmjs.org/
30 silly mapToRegistry uri https://registry.npmjs.org/simple-hl7
31 verbose addNameRange registry:https://registry.npmjs.org/simple-hl7 not in flight; fetching
32 verbose request uri https://registry.npmjs.org/simple-hl7
33 verbose request no auth needed
34 info attempt registry request try #1 at 1:03:15 PM
35 verbose request id b47620dac349376d
36 verbose etag "5ZC1F5GH48GUI13YWVBGH3TSQ"
37 http request GET https://registry.npmjs.org/simple-hl7
38 http 304 https://registry.npmjs.org/simple-hl7
39 verbose headers { date: 'Wed, 30 Dec 2015 12:03:15 GMT',
39 verbose headers   via: '1.1 varnish',
39 verbose headers   'cache-control': 'max-age=300',
39 verbose headers   etag: '"5ZC1F5GH48GUI13YWVBGH3TSQ"',
39 verbose headers   age: '0',
39 verbose headers   connection: 'keep-alive',
39 verbose headers   'x-served-by': 'cache-lhr6334-LHR',
39 verbose headers   'x-cache': 'MISS',
39 verbose headers   'x-cache-hits': '0',
39 verbose headers   'x-timer': 'S1451476995.295379,VS0,VE168',
39 verbose headers   vary: 'Accept' }
40 silly get cb [ 304,
40 silly get   { date: 'Wed, 30 Dec 2015 12:03:15 GMT',
40 silly get     via: '1.1 varnish',
40 silly get     'cache-control': 'max-age=300',
40 silly get     etag: '"5ZC1F5GH48GUI13YWVBGH3TSQ"',
40 silly get     age: '0',
40 silly get     connection: 'keep-alive',
40 silly get     'x-served-by': 'cache-lhr6334-LHR',
40 silly get     'x-cache': 'MISS',
40 silly get     'x-cache-hits': '0',
40 silly get     'x-timer': 'S1451476995.295379,VS0,VE168',
40 silly get     vary: 'Accept' } ]
41 verbose etag https://registry.npmjs.org/simple-hl7 from cache
42 verbose get saving simple-hl7 to /Users/tm/.npm/registry.npmjs.org/simple-hl7/.cache.json
43 silly addNameRange number 2 { name: 'simple-hl7', range: '>=1.4.2 <2.0.0', hasData: true }
44 silly addNameRange versions [ 'simple-hl7',
44 silly addNameRange   [ '0.1.0',
44 silly addNameRange     '0.2.0',
44 silly addNameRange     '0.3.0',
44 silly addNameRange     '0.4.0',
44 silly addNameRange     '1.0.0',
44 silly addNameRange     '1.0.1',
44 silly addNameRange     '1.0.2',
44 silly addNameRange     '1.1.0',
44 silly addNameRange     '1.1.1',
44 silly addNameRange     '1.1.2',
44 silly addNameRange     '1.2.0',
44 silly addNameRange     '1.3.0',
44 silly addNameRange     '1.3.1',
44 silly addNameRange     '1.3.2',
44 silly addNameRange     '1.4.0',
44 silly addNameRange     '1.4.1',
44 silly addNameRange     '1.4.2',
44 silly addNameRange     '1.4.3' ] ]
45 silly addNamed simple-hl7@1.4.3
46 verbose addNamed "1.4.3" is a plain semver version for simple-hl7
47 silly mapToRegistry name simple-hl7
48 silly mapToRegistry using default registry
49 silly mapToRegistry registry https://registry.npmjs.org/
50 silly mapToRegistry uri https://registry.npmjs.org/simple-hl7
51 verbose addRemoteTarball https://registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz not in flight; adding
52 verbose addRemoteTarball [ 'https://registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz',
52 verbose addRemoteTarball   'bc24c25132a3ca7f4dbdef96dfb9ed511f89746b' ]
53 info retry fetch attempt 1 at 1:03:15 PM
54 info attempt registry request try #1 at 1:03:15 PM
55 http fetch GET https://registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz
56 http fetch 200 https://registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz
57 silly fetchAndShaCheck shasum bc24c25132a3ca7f4dbdef96dfb9ed511f89746b
58 verbose stack Error: shasum check failed for /var/folders/3l/hc52g2s91svgrmb8zzbg4sf00000gv/T/npm-912-138f42cc/registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz
58 verbose stack Expected: bc24c25132a3ca7f4dbdef96dfb9ed511f89746b
58 verbose stack Actual:   4594969f78105fb375e0879c0507ddba9a662141
58 verbose stack From:     https://registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz
58 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/sha/index.js:25:8
58 verbose stack     at ReadStream.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/sha/index.js:72:7)
58 verbose stack     at emitNone (events.js:72:20)
58 verbose stack     at ReadStream.emit (events.js:166:7)
58 verbose stack     at endReadableNT (_stream_readable.js:905:12)
58 verbose stack     at doNTCallback2 (node.js:441:9)
58 verbose stack     at process._tickCallback (node.js:355:17)
59 verbose cwd /Users/tm/Code/clicrdv-gcal
60 error Darwin 15.2.0
61 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "simple-hl7"
62 error node v4.2.4
63 error npm  v2.14.12
64 error shasum check failed for /var/folders/3l/hc52g2s91svgrmb8zzbg4sf00000gv/T/npm-912-138f42cc/registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz
64 error Expected: bc24c25132a3ca7f4dbdef96dfb9ed511f89746b
64 error Actual:   4594969f78105fb375e0879c0507ddba9a662141
64 error From:     https://registry.npmjs.org/simple-hl7/-/simple-hl7-1.4.3.tgz
65 error If you need help, you may report this error at:
65 error     <https://github.com/npm/npm/issues>
66 verbose exit [ 1, true ]
```

**Comment by maur1th** [comment]
No issue with installing 1.4.2 though

**Comment by hitgeek** [comment]
sorry about that. I tried to set up npm deploy with travis ci, and obviously that broke something.
i think it might be related to this https://github.com/npm/npm/issues/8822
i just published manually published 1.4.4 and that seems to have fixed it.

**Comment by maur1th** [comment]
Indeed, thanks!
